packages/app/obojobo-repository/shared/components/layouts/default.jsx

```diff
@@ -3,6 +3,8 @@ require('./default.scss')
 const React = require('react')
 const Footer = require('./footer')
 const Dedication = require('./dedication')
+const Notification = require('../notification.jsx').default
```

Comment:
After moving the `Notification` component from this file into the `Dashboard` component, this import will be unnecessary.

See suggested changes for the `notification.jsx` file, which would allow you to `require` this the usual way instead of specifying `.default`.